src/utils/components/Consoles/components/CloudInitCredentials/CloudInitCredentials.tsx

```diff
@@ -23,8 +25,15 @@ const CloudInitCredentials: React.FC<CloudInitCredentialsProps> = ({ vmi }) => {
   const { user, password } = getCloudInitCredentials(vmi);
   const [showCredentials, setShowCredentials] = React.useState<boolean>(false);
 
+  if (!user && !password) {
```

Comment:
```suggestion
  if (!user) {
```
this can also cover the case of `(!user && password)`